proposed/MPID Messaging.md

```diff
@@ -396,47 +384,221 @@ impl MpidMessage {
 }
 ```
 
-To send an MPID Message, a client would do something like:
-
-```rust
-let mpid_message = MpidMessage::new(my_mpid: Mpid, recipient: ::routing::Authority::Client,
-                                    metadata: Vec<u8>, body: Vec<u8>);
-
-```
-
 Account types held by MpidManagers
 
 ```rust
-struct Outbox {
+struct OutboxAccount {
     pub sender: ::routing::NameType,
+    pub sender_clients: Vec<::routing::Authority::Client>,
```

Comment:
when MpidManager(A) received a failure from MpidManager(B) (indicates Inbox full), it needs to notify the client, which is not a response (as the ori_request has already been forwarded and discarded) but a Vault initiated notification.  :(